lib/everest/slac/fsm/evse/src/states/matching_handle_slac.cpp

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright 2023 - 2023 Pionix GmbH and Contributors to EVerest
+// Copyright 2023 - 2025 Pionix GmbH and Contributors to EVerest
```

Comment:
D'uh, I should have dropped this change or adapted to headers to the recent style.